login.js

```diff
@@ -0,0 +1,81 @@
+const emailInput = document.getElementById("userName");
+const passwordInput = document.getElementById("userPassword");
+
+const errorMessage = document.getElementById("error-message");
+const errorPassword = document.getElementById("error-password");
```

Comment:
💊 제안
이름은 코드에 있어 내용을 파악하는데 도움을 주는 중요한 단서입니다.
지금 errorMessage의 경우는 이메일관련 에러를 나타내는 태그의 클래스명인데 일반 명사인 errorMessage로 이름이 지어져 있어
errorPassword와 비교해서 어떤 변수인지 명확하지 않습니다.
가능하면 어떤 변수인지 잘 설명해줄 수 있는 이름으로 변경해주세요~
```suggestion
const errorEmail = document.getElementById("error-message");
const errorPassword = document.getElementById("error-password");
```